client/reader/tag-stream/main.jsx

```diff
@@ -120,7 +120,7 @@ class TagStream extends Component {
 					<ReaderBackButton />
 					<TagStreamHeader
 						title={ title }
-						imageSearchString={ imageSearchString }
```

Comment:
`imageSearchString` was not getting used inside the component and using `encodedTagSlug` makes most sense instead.